### PR TITLE
Update fees and rentals_fees endpoints with new actions and attributes

### PR DESCRIPTION
**Pull request by Fredar**

### CODE REVIEW

File: api_calls/reference/fees/create-request.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "fees": [
+    {
+      "name":{
```
Comment (ZenCocoon) [inline api_calls/reference/fees/create-request.json]: On write, it need to use `name_en` instead.

File: api_calls/reference/rentals_fees/create-request.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "rentals_fees": [
+    {
+      "fee_id": 9,
+      "always_applied": true,
+      "maximum_bookable": 12
```
Comment (ZenCocoon) [inline api_calls/reference/rentals_fees/create-request.json]: No need to define if `public`, `private` or `required`?

File: content/reference/endpoints/fees.md [hunk 3]
```diff
@@ -44,4 +62,36 @@ Returns a single fee identified by ID.
...
+~~~
+PUT /fees/:fee_id
+~~~
+
+<%= render 'json_response', endpoint: "fees", request: "update",
+  scopes: [{ rates_write: "public-rates_write" }] %>
+
+## Destroy a fee
+
+Required OAuth scope: `:rates_write`
```
Comment (ZenCocoon) [inline content/reference/endpoints/fees.md]: I think good to explain that it's really getting archived and looking at `archived_at` is the way to identify if a `fee` should not longer be considered as valid.

File: content/reference/endpoints/rentals_fees.md [hunk 4]
```diff
@@ -16,15 +17,41 @@ id               | Integer | Read       | Rentals Fee's id.
...
+fee_id           | Integer | Write      | Fee id related to the Rentals Fee.
+season_ids       | Array   | Write      | Season ids related to the Rentals Fee.
+-----------------|---------|------------|------------
+always_applied   | Boolean | Read/Write | Available to all seasons and periods if set to true.
+maximum_bookable | Integer | Read/Write | Rentals Fee's maximum booked count.
+name             | [Object](/reference/enums#formats)   | Read       | Rentals Fee's name, list of locales described in [enums section](/reference/enums#locales).
+public           | Boolean | Read       | Publicly bookable by client if set to true.
+rate             | [Decimal](/reference/enums#formats)  | Read       | Rentals Fee's rate.
+rate_kind        | String  | Read       | Rentals Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
+required         | Boolean | Read       | Always included for new bookings, also public if set to true.
```
Comment (ZenCocoon) [inline content/reference/endpoints/rentals_fees.md]: this should be writtable

File: content/reference/endpoints/rentals_fees.md [hunk 5]
```diff
@@ -16,15 +17,41 @@ id               | Integer | Read       | Rentals Fee's id.
...
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Rentals Fee's id.
+fee_id           | Integer | Write      | Fee id related to the Rentals Fee.
+season_ids       | Array   | Write      | Season ids related to the Rentals Fee.
+-----------------|---------|------------|------------
+always_applied   | Boolean | Read/Write | Available to all seasons and periods if set to true.
+maximum_bookable | Integer | Read/Write | Rentals Fee's maximum booked count.
+name             | [Object](/reference/enums#formats)   | Read       | Rentals Fee's name, list of locales described in [enums section](/reference/enums#locales).
+public           | Boolean | Read       | Publicly bookable by client if set to true.
```
Comment (ZenCocoon) [inline content/reference/endpoints/rentals_fees.md]: this should be writtable

File: content/reference/endpoints/rentals_fees.md [hunk 6]
```diff
@@ -16,15 +17,41 @@ id               | Integer | Read       | Rentals Fee's id.
...
+rate             | [Decimal](/reference/enums#formats)  | Read       | Rentals Fee's rate.
+rate_kind        | String  | Read       | Rentals Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
+required         | Boolean | Read       | Always included for new bookings, also public if set to true.
+-----------------|---------|------------|------------
```
Comment (ZenCocoon) [inline content/reference/endpoints/rentals_fees.md]: Missing `private`, which is writable as well

File: content/reference/endpoints/fees.md (hunk 3 above)
Comment (Fredar) [inline content/reference/endpoints/fees.md]: Why do we even return archived fees?

File: content/reference/endpoints/rentals_fees.md (hunk 5 above)
Comment (Fredar) [inline content/reference/endpoints/rentals_fees.md]: its not on core

File: content/reference/endpoints/rentals_fees.md (hunk 4 above)
Comment (Fredar) [inline content/reference/endpoints/rentals_fees.md]: its not on core

File: content/reference/endpoints/rentals_fees.md (hunk 6 above)
Comment (Fredar) [inline content/reference/endpoints/rentals_fees.md]: its not on core

File: content/reference/endpoints/fees.md (hunk 3 above)
Comment (ZenCocoon) [inline content/reference/endpoints/fees.md]: cause we might still have links to them from `bookings_fees`.

File: content/reference/endpoints/rentals_fees.md (hunk 5 above)
Comment (ZenCocoon) [inline content/reference/endpoints/rentals_fees.md]: on, on admin v2 it's using `status`, should be here as well then. Needed to set this.

Comment (Fredar) [inline content/reference/endpoints/rentals_fees.md]: status added

**Pull request merged**